meteomatics/api.py

```diff
@@ -69,12 +70,24 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def handle_proxy(func):
+    @wraps(func)
+    def wrapper(*args, **kwargs):
+        if not len(Config.get("PROXIES")) == 0:
```

Comment:
maybe add a comment along the lines of "passing the proxy dict to the requests proxies kwarg"